meson_options.txt

```diff
@@ -2,3 +2,8 @@ option('tests',
 	type: 'boolean',
 	value: true,
 	description: 'Run the tests (default=yes)')
+
+option('runtime_depcheck',
```

Comment:
`runtime-dependency-checks` - less ambiguous and using underscores in options is generally frowned upon